src/jmuxer.js

```diff
@@ -288,18 +313,39 @@ export default class JMuxer extends Event {
     }
 
     startInterval() {
-        this.interval = setInterval(()=>{
+        this.interval = setInterval(() => {
             if (this.bufferControllers) {
                 this.releaseBuffer();
                 this.clearBuffer();
             }
         }, this.options.flushingTime);
     }
 
+    startSeekInterval() {
```

Comment:
Redundant, we can the same interval instead of creating the same interval with a different name